lib/omnibus-ctl.rb

```diff
@@ -506,6 +512,67 @@ def reconfigure(exit_on_success=true)
       end
     end
 
+    def check_license_acceptance(override_accept = false)
+      license_guard_file_path = File.join(data_path, ".lic.accepted")
```

Comment:
Why .lic.accepted for the license accepted file? Why not spell out license, so .license.accepted? I'm generally in favor of being as clear as possible, and it's just a few characters that make it more clear what the file is for.